### PR TITLE
Melzer/hana data disk configuration documentation / solve issue #622

### PR DESCRIPTION
**Pull request by melzer-b1**
This merge will add a user documentation, how to use the complex variable hana_data_disks_configuration in azure.

### CODE REVIEW

File: azure/README.md [hunk 1]
```diff
@@ -131,11 +131,80 @@ Netweaver virtual ips: 10.74.0.34, 10.74.0.35, 10.74.0.36, 10.74.0.37
 
 ## HANA configuration
 
+### HANA data disks configuration
+
+The whole disk configuration is made in on variable named `hana_data_disks_configuration` by defining a map. It looks overwhelming at first because it encapsulates hard disk selection, logical volumes and data destinations. During deployment the HANA VM will expect a standard set of directories for its data `/hana/data`, `/hana/log`, `/hana/shared`, `/usr/sap` and `/hana/backup`. Some systems also use `/sapmnt`. We will go through it step by step.
```
Comment (arbulu89) [inline azure/README.md:136]: It looks something is missing in `.. is made in on variable..```

Comment (arbulu89) [reply azure/README.md:136]: I personally would remove the `It looks overhelming at first because` text.

Comment (arbulu89) [reply azure/README.md:136]: Try to not use `will`. Use the present time. `During the deployment, HANA VMs expeect a standard set...`

**Review by arbulu89 — CHANGES_REQUESTED** [review]
Hi @melzer-b1 ,
The content looks great.
I suggest to change a bit the writting style. 
- Use the present time (replace all these `will` cases)
- Try to avoid the `we` usage. Try to talk about the project perspective (this might not be possible always)

**Comment by melzer-b1** [comment]
I've made the changes as purposed and also updated phrases in other parts of the documentation for consistency.

**Review by arbulu89 — COMMENTED** [review]
Hey @melzer-b1 ,
As a comment, I see multiple commits with the same text.
It would make sense if you just amend the changes in this case, rather than creating a new commit.
If we use the same text, I assume that the change belongs to the same logic, so it should be better to have them in the same commit

**Comment by melzer-b1** [comment]
rebased to limit number of commits

File: azure/README.md [hunk 2]
```diff
@@ -131,11 +140,78 @@ Netweaver virtual ips: 10.74.0.34, 10.74.0.35, 10.74.0.36, 10.74.0.37
...
+
+ | Parameter     | VG1        | VG2       | VG3          | VG4      | VG5          |
+ | ---------     | ---        | ---       | ---          | ---      | ---          |
+ | **names**     | data       | log       | shared       | usrsap   | backup       |
+ | **luns**      | 0,1        | 2,3       | 4            | 5        | 6            |
+ | **lv_sizes**  | 100        | 100       | 1000         | 100      | 100          |
+ | **paths**     | /hana/data | /hana/log | /hana/shared | /usr/sap | /hana/backup |
+
+As you see, there are 5 volume groups specified. Each volume group has it own name. It is set with parameter `names`.  The parameter `luns` assigns one LUN or a combination of several LUNs to a volume group. In the example above `data` is uses disk with LUN **0** and **1**, but `backup` only uses disk with LUN **6**. A LUN can only assigned to one volume group.
```
Comment (arbulu89) [inline azure/README.md:180]: Typo `... has its onw name..`

Comment (arbulu89) [reply azure/README.md:180]: Typo in `data is uses` -> `data uses`

Comment (arbulu89) [reply azure/README.md:180]: Typo `only be assigned`

File: azure/README.md [hunk 3]
```diff
@@ -131,11 +140,78 @@ Netweaver virtual ips: 10.74.0.34, 10.74.0.35, 10.74.0.36, 10.74.0.37
...
+
+Using the example above for volume group `data` again to show how a HANA VM is effected. As said the `data` volume group uses two physical disks. They are used as physical volume (i. e. `/dev/sdc` and `/dev/sdd` resp. LUN **0** and **1**). Both physical volumes share the same volume group named `vg_hana_data`. A logical volume named `lv_hana_data_0` allocates **100%** of this volume group. The logical volume name is generated from the volume group name. The logical volume is mounted at mount point `/hana/data`.
```
Comment (arbulu89) [inline azure/README.md:182]: `effected`? Maybe it was affected

File: azure/README.md [hunk 4]
```diff
@@ -131,11 +140,78 @@ Netweaver virtual ips: 10.74.0.34, 10.74.0.35, 10.74.0.36, 10.74.0.37
 
 ## HANA configuration
 
+### HANA data disks configuration
+
+The whole disk configuration is made by configuring a variable named `hana_data_disks_configuration`. It looks complex, because it encapsulates hard disk selection, logical volumes and data destinations in a compact form. This section describes all parameters line by line.
```
Comment (arbulu89) [inline azure/README.md:145]: Remove the `It looks complex`. It looks like we are putting some excuses. This all text is to explain so it is not complex anymore

File: azure/README.md [hunk 5]
```diff
@@ -131,11 +140,78 @@ Netweaver virtual ips: 10.74.0.34, 10.74.0.35, 10.74.0.36, 10.74.0.37
...
+
+In Detail: `disks_type` selects the sort of SSD with bandwidth and redundancy options. You find possible selections and costs at [Azure – Managed Disks](https://azure.microsoft.com/en-us/pricing/details/managed-disks/).
+The parameter `disks_size` selects the size of each disk in GB. Also you can decide if `caching` or the `writeaccelerator` is active.
+
+The disks are counted from left to right beginning with **0**. This number is called LUN. A Logical Unit Number (LUN) is a SCSI concept for logical abstraction targeting physical drives. If you have 5 disks you count **0,1,2,3,4**.
+
+After describing the physical disks, the logical volumes can be specified using the parameters `luns`, `names`, `lv_sizes` and `paths`. These lines look confusing, caused by the usage of the `#` signs and commas. The comma combines several values into one value and the `#` sign is used for separation of volume groups. Think about the `#` sign as a column separator in a table then it will look like:
```
Comment (arbulu89) [inline azure/README.md:171]: Remove the `These lines look confusing`.  I don't like this text, it kind of says `yeah, it is not that good..` hehe

**Review by arbulu89 — APPROVED** [review]
Hey @melzer-b1 ,
I'm approving the PR. There are some minor details you would need to fix.
But I don't see we need more reviews.
Nice job

**Pull request merged**